### PR TITLE
Updated coding style of posted IPI implementation

### PR DESCRIPTION
**Pull request by jackhumphries**

### CODE REVIEW

File: libdune/entry.c [hunk 1]
```diff
@@ -661,13 +661,17 @@ int dune_init(bool map_full)
...
+	if (!dune_setup_apic()) {
+		printf("dune: could not set up APIC\n");
+		ret = -ENOMEM;
+		goto err;
+	}
 
 	return 0;
 
 err:
 	// FIXME: need to free memory
```
Comment (kkaffes) [inline libdune/entry.c:671]: Do you still need the FIXME here?

File: libdune/apic.c [hunk 2]
```diff
@@ -20,52 +21,68 @@
...
-        }
+		if (error) *error = true;
+		return 0;
+	}
 	return apic_routing[cpu];
 }
 
 static inline unsigned int __prepare_ICR(unsigned int shortcut, int vector,
-                                         unsigned int dest)
+										 unsigned int dest)
```
Comment (kkaffes) [inline libdune/apic.c]: Fix indentation.

Comment (jackhumphries) [reply libdune/apic.c]: Ugh ok I'll get that, didn't notice it

File: libdune/entry.c (hunk 1 above)
Comment (jackhumphries) [reply libdune/entry.c:671]: That was there originally... it must be a reference to something else

File: kern/vmx.c [hunk 3]
```diff
@@ -70,12 +70,12 @@ static DECLARE_BITMAP(vmx_vpid_bitmap, VMX_NR_VPIDS);
...
 typedef struct posted_interrupt_desc {
-    u32 vectors[8]; /* posted interrupt vectors */
-    u32 extra[8]; /* outstanding notification indicator and extra space the VMM can use */
+	u32 vectors[8];	/* posted interrupt vectors */
+	u32 extra[8];	/* outstanding notification indicator and extra space the VMM can use */
 } __aligned(64) posted_interrupt_desc;
 
 static unsigned long *msr_bitmap;
-//static struct vmx_vcpu *vcpus_hash[NR_CPUS];
+static struct vmx_vcpu *vcpus_hash[NR_CPUS];
```
Comment (kkaffes) [inline kern/vmx.c:78]: What is the use of this struct?

Comment (jackhumphries) [reply kern/vmx.c:78]: That was originally present in Dune... I commented it out by mistake when implementing IPIs

Comment (jackhumphries) [reply kern/vmx.c:78]: It's a hash table that maps physical core ID to vcpu

**Pull request merged**